package.json

```diff
@@ -46,6 +46,7 @@
     "@opentelemetry/semantic-conventions": "~1.21.0",
     "@typescript-eslint/eslint-plugin": "^6.20.0",
     "@typescript-eslint/parser": "^6.20.0",
+    "axios": "^1.6.7",
```

Comment:
debated whether to avoid adding an additional package to get this functionality... open to thoughts